scripts/core/editor3/helpers/highlights.js

```diff
@@ -262,12 +262,16 @@ export function getHighlightDataAtOffset(editorState, types, selection, offset)
  */
 export function addHighlight(editorState, type, data) {
     const highlightsState = getHighlightsState(editorState);
-    const styleName = type + '-' + (highlightsState.lastHighlightIds[type] + 1);
+    let newIndex = 0;
+    if (highlightsState.lastHighlightIds && _.has(highlightsState.lastHighlightIds, type)) {
+        newIndex = highlightsState.lastHighlightIds[type] + 1;
+    }
+    const styleName = type + '-' + newIndex;
 
     const newHighlightsState = {
         lastHighlightIds: {
             ...highlightsState.lastHighlightIds,
-            [type]: highlightsState.lastHighlightIds[type] + 1
+            [type]: newIndex
```

Comment:
Is this change only to check for null/undefined values? `highlightsState.lastHighlightIds[type]` should never be null/undefined by design, so check might not be required